graylog2-web-interface/src/components/sidecars/failure-tracking/SidecarFailureTrackingList.tsx

```diff
@@ -0,0 +1,196 @@
+
+import VerboseMessageModal from '../sidecars/VerboseMessageModal';
+import type { Collector, PaginationInfo, SidecarSummary } from '../types';
+
+const StyledSortIcon = styled(SortIcon)`
+  && {
+    width: 12px;
+    margin-left: 5px;
+    line-height: 1;
+  }
+`;
+
+const StandardWidthCol = styled.col`
+  width: 9%;
+`;
+
+const ErrorMessageCol = styled.col`
+  width: 14%;
+`;
+
+const VerboseMessageCol = styled.col`
+  width: 50%;
+`;
+
+type Props = {
+  sidecars: SidecarSummary[],
+  collectors: Collector[],
+  pagination: PaginationInfo,
+  query: string,
+  sort: { field: string, order: string },
+  onlyActive: boolean,
+  pageSizes: number[],
+  onPageChange: (page: number, pageSize: number) => void,
+  onQueryChange: (query: string) => void,
+  onSortChange: (sortField: string) => void,
+  toggleShowInactive: () => void,
+}
+
+const SidecarFailureTrackingList = ({
+  sidecars,
+  collectors,
+  pagination,
+  query,
+  sort,
+  onlyActive,
+  pageSizes,
+  onPageChange,
+  onQueryChange,
+  onSortChange,
+  toggleShowInactive,
+}: Props) => {
+  const [collectorDetailsToShow, setCollectorDetailsToShow] = useState<{ name: string, verbose_message: string }|null>(null);
+
+  const formatSidecarList = (sidecarRows: React.ReactNode[]) => {
+    const sidecarCollection = {
+      node_name: 'Sidecar',
+      collector: 'Collector',
+      last_seen: 'Last Seen',
+      'node_details.status.status': 'Status',
+      message: 'Error Message',
+      verbose_message: 'Verbose Message',
+    };
+
+    return (
+      <Table striped responsive>
+        <colgroup>
+          <StandardWidthCol />
+          <StandardWidthCol />
+          <StandardWidthCol />
+          <StandardWidthCol />
+          <ErrorMessageCol />
+          <VerboseMessageCol />
+        </colgroup>
+        <thead>
+          <tr>
+            {Object.keys(sidecarCollection).map((sort_key) => (
+              <th key={sort_key}>
+                {sidecarCollection[sort_key]}
+                {['node_name', 'last_seen'].includes(sort_key) && (
+                  <StyledSortIcon activeDirection={sort.field === sort_key ? sort.order : null}
+                                  onChange={() => onSortChange(sort_key)}
+                                  title={sort_key}
+                                  ascId="asc"
+                                  descId="desc" />
+                )}
+              </th>
+            ))}
+          </tr>
+        </thead>
+        <tbody>
+          {sidecarRows}
+        </tbody>
+      </Table>
+    );
+  };
+
+  const formatNoMatchingListAlert = () => {
+    const showInactiveHint = (onlyActive ? ' and/or click on "Include inactive sidecars"' : null);
+
+    return (
+      <NoSearchResult>
+        {`There are no sidecars with failures matching the search criteria. Try adjusting your search filter${showInactiveHint}.`}
+      </NoSearchResult>
+    );
+  };
+
+  const renderEmptyList = () => {
```

Comment:
Again, could be changed to a comp:

```suggestion
  const EmptyList = () => {
```